tap_bitbucket/gitlocal.py

```diff
@@ -53,10 +56,18 @@ def hashPatchLine(patchLine, hmacToken = None):
 
 
 def parseDiffLines(lines, shouldEncrypt=False, hmacToken=None):
+  MAX_DIFF_LINES=1000
```

Comment:
These were just some off the cuff numbers.  They seem pretty permissive in my opinion.  I would have to guess 90% of code would fall inside these constraints.

Reply:
Let's do 10k lines for the diff since some files are that big.